### PR TITLE
PgBouncer does support TLS, update documentation

### PR DESCRIPTION
**Pull request by andreasscherbaum**

### CODE REVIEW

**Comment by mkiyama** [comment]
Adding @litzell who worked on pgBouncer

Is this for GPDB 5.0 only?
Is GPDB using a newer version of pgBouncer?

Chuck -  do we mention the version of pgBouncer somewhere?

**Comment by ascherbaum-pivotal** [comment]
This should be true for 4.3 and 5.0, because pgBouncer is an external tool.
It also depends which version we build - but I hope it's a recent one.

**Comment by mkiyama** [comment]
@lij55 
Can you comment on support on TLS support in the version of pgBouncer included with GPDB 4.3.x and 5.0?

**Comment by litzell** [comment]
The docs were last updated for pgbouncer 1.6.1. If the code is updated, we need to check the pgbouncer release notes for additional changes.

**Comment by litzell** [comment]
GPDB has pgbouncer 1.6.1, which doesn't have TLS support. We have added an LDAP feature for GPDB to this version, so the GPDB version is not the same as the external code.

GPDB 5.0 alpha 5:

[gpadmin@gpdb5a5 ~]$ pgbouncer --version
pgbouncer version 1.6.1 (compiled by <root@b0d3dc90-76cb-43a4-7950-446d6dd4d46c> at 2017-05-19 18:16:15)

**Comment by lij55** [comment]
The bundled pgbouncer doesn't support TLS/SSL currently. Recommended security is to use stunnel, which is tested with GPDB before.
We are working on the pgbouncer upgrading.

File: gpdb-doc/dita/admin_guide/access_db/topics/pgbouncer.xml [hunk 1]
```diff
@@ -76,11 +76,13 @@
           manage PgBouncer using SQL-like commands. Configuration parameters can be changed without
           having to restart PgBouncer, and the configuration file can be reloaded to pick up
           changes.</p>
-        <p>PgBouncer does not yet support SSL connections. If you want to encrypt traffic between
-          clients and PgBouncer, you can use <xref href="https://www.stunnel.org" format="html"
-            scope="external">stunnel</xref>, a free software utility that creates TLS-encrypted
-          tunnels using the OpenSSL cryptography library. See <xref href="#topic_nfn_tvd_5s"
-            format="dita"/> for directions.</p>
+        <p>PgBouncer does support TLS connections. Details how to configure TLS are available on th
```
Comment (danielgustafsson) [inline gpdb-doc/dita/admin_guide/access_db/topics/pgbouncer.xml]: s/th/the/

**Comment by ascherbaum-pivotal** [comment]
Looking at the changelog, I don't see any imminent reasons to upgrade, but having TLS would be nice indeed. It's there for 1.5 years now, and will make configuration more easy.

http://pgbouncer.github.io/changelog.html

Guess I retract that PR for now. @lij55 Is there any timeline for the upgrade?

**Comment by lij55** [comment]
Hi @andreasscherbaum , 

Client team is working on the upgrade now.

**Comment by ascherbaum-pivotal** [comment]
@lij55 Do you have an update available, or a timeline?

**Comment by lij55** [comment]
Hi @ascherbaum-pivotal ,
We already start this job(https://github.com/yuanzhao/gpdb/commit/34c89fc7815410b3b4de4a97fb085fb4650c041e) and can't ensure when to finish.
Do you have any emergency requirement?

**Comment by andreasscherbaum** [comment]
@lij55 Nothing special - just thinking if it's worth keeping this PR open.

**Comment by lij55** [comment]
No problem. We will keep it updated if we have progress.

**Comment by Eulerizeit** [comment]
What is happening with this PR?

**Comment by andreasscherbaum** [comment]
Not sure if PgBouncer was updated in the meantime.

**Comment by Eulerizeit** [comment]
Why don't we close this PR and then change the docs when pgBouncer is updated

**Comment by andreasscherbaum** [comment]
See discussion around End of June, beginning July. It looked like the version will be upgraded "soon". Looks like that did not happen.

**Comment by lij55** [comment]
I also suggest we update the doc when new pgbouncer is ready.

**Pull request closed without merging**